extensions/sql-migration/src/constants/strings.ts

```diff
@@ -270,6 +285,7 @@ export const SUMMARY_AZURE_STORAGE = localize('sql.migration.summary.azure.stora
 export const SUMMARY_IR_NODE = localize('sql.migration.ir.node', "Integration Runtime node");
 export const NETWORK_SHARE = localize('sql.migration.network.share', "Network Share");
 export const BLOB_CONTAINER = localize('sql.migration.blob.container.title', "Blob Container");
+export const BLOB_CONTAINER_LAST_BACKUP_FILE = localize('sql.migration.blob.container.last.backup.file.label', "Last Backup File");
```

Comment:
for label of controls, only the first letter of the first word needs be upper case. It would be good if you can clean up the places that are not following this convention.

Reply:
Next week, we will be revising the strings based on feedback from out content writer. We will address this then.
cc: @brian-harris